manifests/prometheus.yaml

```diff
@@ -0,0 +1,65 @@
+# This manifests creates a minimal Prometheus server to scrape and display the metrics emitted by the workflow
+# controller. To open this server, create an external IP for the prometheus service or use kubectl port-forward,
+# then open:
+#   
+#   localhost:9091/graph
+#
+# Note: this assumes the workflow-controller is emitting metrics in the default port (9090). This will need to
+# be modified if the default is overriden.
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: prometheus
+  name: prometheus
+spec:
+  ports:
+  - name: prometheus
+    port: 9091
+    protocol: TCP
+    targetPort: 9090
+  selector:
+    app: prometheus
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-config-cm
+data:
+  prometheus.yaml: |
+    global:
+      scrape_interval: 15s
+    scrape_configs:
+      - job_name: 'argo'
+        static_configs:
+          - targets: ['workflow-controller-metrics:9090']
```

Comment:
add localhost:9090

Reply:
I think I'm wrong about this - I seem to have problems with both localhost and this

Reply:
Testing right now

Reply:
Had a small mistake, should work now